vue2-vuetify/src/controls/DateControlRenderer.vue

```diff
@@ -220,12 +222,17 @@ const controlRenderer = defineComponent({
       const date = parseDateTime(value, this.formats);
       return date ? date.format(this.dateFormat) : value;
     },
-    pickerValue(): string | undefined {
-      const value = this.control.data;
+    pickerValue: {
+      get(): string | undefined {
+        const value = this.control.data;
 
-      const date = parseDateTime(value, this.formats);
-      // show only valid values
-      return date ? date.format('YYYY-MM-DD') : undefined;
+        const date = parseDateTime(value, this.formats);
+        // show only valid values
+        return date ? date.format('YYYY-MM-DD') : undefined;
+      },
+      set(val: string): void {
+        this.onPickerChange(val);
+      },
```

Comment:
Nice pattern to use the `v-model` approach. Not worth to change it now but we could use the same pattern for all inputs actually.